set.mm

```diff
@@ -501875,6 +501875,69 @@ it has no (non-trivial) cycles.  (Contributed by BTernaryTau,
       NUOUPUQURUS $.
   $}
 
+  ${
+    $d f g p $.  $d f G p $.  $d f V p $.
```

Comment:
The class variables are usually either at the beginning or at the end. I do not know any case where they are in the middle of the sequence of setvar variables in a DV condition.

Reply:
I've just been copying the distinct variable conditions from mm-lamp, so all of my theorems use whatever order it uses. I don't see any mention of this in https://us.metamath.org/mpeuni/conventions.html, so either I'm missing something (definitely possible) or this isn't documented. Either way, that's probably why mm-lamp doesn't respect it. If you want, I can create a separate PR to update all my distinct variable conditions, and also file a feature request for mm-lamp to make sure other users don't run into this issue.

Reply:
For the record, @avekens has [posted on the metamath mailing list](https://groups.google.com/g/metamath/c/vRXuz7Qs5wY) on the topic.
I agree with @digama0's answer that users should not worry about order of the DV conditions, i.e. tools should handle it transparently for us.

Reply:
According to @digama0's "class/wff variables should come at the beginning", this should be done manually, until the tools take care for it. This holds already for the cases in this PR. The setvar variables should be ordered alphabetically.

Reply:
All I see regarding manual ordering is:
> But also, this is not a thing that users should ever have to think about.

I'd prefer to wait to make any changes until a consensus has been reached regarding placing class/wff vars at the beginning vs at the end of DV conditions.

Reply:
I agree with @BTernaryTau , until we have formatting support let's just not worry about it, it won't make the formatter PR any different.

Reply:
For now, it can remain as it is, because it is in a mathbox. I do not know how long it takes until a formatter is availabe, and at least the main body of set.mm should not contain additional variants of ordering the variables in a DV statement. I plan to add a correspondig convention in section "Conventions" soon.

Reply:
To put it another way: until we have a formatter, I think the convention should be that the $d declarations are in arbitrary order, and if people want guidance on what order to write things in we have a suggestion but there is no requirement that they abide by it, in main or otherwise.